sdk/test-utils/test-utils-vitest/README.md

```diff
@@ -0,0 +1,254 @@
+# Azure test-utils client library for JavaScript
+
+The Azure SDK for JavaScript is composed of a multitude of repositories that attempt to deliver a common, homogenous SDK to make use of all the services that Azure can provide.
+
+This non-shipping library `@azure/test-utils-vitest` attempts to add additional testing support to libraries. It is supposed to be added only as a devDependency and should be used only for the tests of an SDK library.
+
+## Getting started
+
+We're about to go through how to set up your project to use the `@azure/test-utils-vitest` package.
+
+This document assumes familiarity with [git](https://git-scm.com) and [rush](https://rushjs.io).
+You can read more about how we use rush in the following links:
+
+- Rush used for [Project Orchestration](https://github.com/sadasant/azure-sdk-for-js/blob/master/CONTRIBUTING.md#project-orchestration).
+- [Rush for NPM users](https://github.com/sadasant/azure-sdk-for-js/blob/master/CONTRIBUTING.md#rush-for-npm-users).
+
+Keep in mind that `@azure/test-utils-vitest` is not a published package. It is only intended
+to be used by the libraries in the `azure-sdk-for-js` repository.
+
+### Installing the package
+
+To install the `@azure/test-utils-vitest` package, you'll need to start by cloning our
+`azure-sdk-for-js` repository. One way of doing this is by using the git command line interface, as
+follows:
+
+```bash
+cd /path/to/my/github/repositories
+git clone https://github.com/Azure/azure-sdk-for-js/
+```
+
+Having cloned this repository, let's set it up by running the following rush commands:
+
+```bash
+cd azure-sdk-for-js
+rush update
+rush install
```

Comment:
Just one of `rush update` `rush install` should be enough. We should have similar information somewhere already? maybe also link to that dev/test or contributing doc.

Reply:
Just used rush install